README.md

```diff
@@ -0,0 +1,37 @@
+
+## Available Scripts
```

Comment:
Do we need such kind of README ? I think we can skip part regarding tests. Do you plan to have 'tests' for this app?